### ISSUE TITLE
Looks like you installed react-native globally, maybe you meant react-native-cli?

### ISSUE THREAD

**Issue by az-ericso** [yuanyan/react-native-web-example#7]
Unable to run the app.

``` sh
usc02qf5yyg8wp:react-native-web-example kklh716$ npm start

> react-native-web-example@0.0.1 start /Users/kklh716/az/react-native-web-example
> react-native start

Looks like you installed react-native globally, maybe you meant react-native-cli?
To fix the issue, run:
npm uninstall -g react-native
npm install -g react-native-cli
```

Followed the instruction above to clean up the react-native but still seeing the same error.

**Comment by j0xhn** [comment]
yeah, I got the same error -- I just looked in the package.json file and saw it was doing a `react-native start` command.  I do it from outside of npm and it worked :/ hopefully that help you @az-ericso

**Comment by oklas** [comment]
Same error only just execute `react-native`

> ```sh
>  D:\1\proj\node_modules\.bin>react-native
>  Looks like you installed react-native globally, maybe you meant react-native-cli?
>  To fix the issue, run:
>  npm uninstall -g react-native
>  npm install -g react-native-cli
> ```

**Comment by hani647** [comment]
I also ran into same issue, as an alternative install this package with yarn globally i.e.
First remove package
`sudo npm uninstall -g react-native`
then install with yarn
`sudo yarn global add react-native-cli`
This worked for me !
